src/components/safety/gallery-data.js

```diff
@@ -7,7 +7,7 @@ module.exports = [
     "video": "cIPpBLTC74c",
     "tips": [
-      "Protect your logins. Use a <a href='https://www.wired.com/2016/01/you-need-a-password-manager/'>password manager</a> to make creating and remembering passwords easier. (Note: we haven’t tried them all – see what works for you.)"
+      "Protect your logins. Use a password manager (like the default <a href='https://www.mozilla.org/en-US/firefox/features/password-manager/?utm_campaign=safety&utm_medium=referral&utm_source=advocacy.mozilla.org'>password manager in Firefox</a>) to make creating and remembering passwords easier. Check out <a href=''https://www.wired.com/2016/01/you-need-a-password-manager/>this list from Wired</a> for more options. (Note: we haven’t tried them all – see what works for you.)"
```

Comment:
I just noticed your second link is a little busted. Not biggie, just looks like a type:

`<a href=''https://www.wired.com/2016/01/you-need-a-password-manager/>this list from Wired</a>`

Looks like the quotes on the `href` are missing the link target.